CoreFoundation/URL.subproj/CFURLPriv.h

```diff
@@ -432,6 +432,12 @@ typedef unsigned long long CFURLResourcePropertyFlags;
 CF_EXPORT
 Boolean _CFURLGetResourcePropertyFlags(CFURLRef url, CFURLResourcePropertyFlags mask, CFURLResourcePropertyFlags *flags, CFErrorRef *error) API_AVAILABLE(macos(10.6), ios(4.0), watchos(2.0), tvos(9.0));
 
+CF_EXPORT
+Boolean __CFURLResourceValueHasFlag(CFURLRef url, CFURLResourcePropertyFlags key, errno_t *error);
+CF_EXPORT
+Boolean _CFURLResourceValueSetFlag(CFURLRef url, CFURLResourcePropertyFlags key, Boolean set, errno_t *error);
+
```

Comment:
I think these should be in `ForSwiftFoundationOnly` and decorated with `CF_SWIFT_EXPORT`, correct?

Reply:
@parkera I applied your suggestions and have done some refactoring and enhancements.